kong-ngx-build

```diff
@@ -11,6 +11,7 @@ LUAROCKS_VER=
 BUILD_CACHE=1
 ARTIFACT_CACHE=1
 OPENRESTY_PATCHES=1
+RESTY_PCRE_VER=8.41
```

Comment:
This can be called simply `PCRE_VER`, and it can default to empty (like the `LUAROCKS_VER` one) so if user does not specify the option in command line we still use the system default (which may help cases like testing and developing).